develop/migration_mt_django.rst

```diff
@@ -5,76 +5,185 @@
 Django
 ------
 
-In :ref:`mt_schema_migration` we discussed the framework-agnostic database changes required for using Citus in the multi-tenant use case. Here we investigate specifically how to migrate multi-tenant Django applications to a Citus storage backend.
+In :ref:`mt_schema_migration` we discussed the framework-agnostic database changes required for using Citus in the multi-tenant use case. Here we investigate specifically how to migrate multi-tenant Django applications to a Citus storage backend with the help of the `django-multitenant <https://github.com/citusdata/django-multitenant>`_ library.
+
+
+
+This process will be in 5 steps:
+
+- Introducing the tenant column to models missing it that we want to distribute
+- Changing the primary keys of distributed tables to include the tenant column
+- Updating the models to use the :code:`TenantModelMixin`
+- Distributing the data
+- Updating the Django Application to scope queries
+
 
 Preparing to scale-out a multi-tenant application
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Initially you’ll start with all tenants placed on a single database node. Django's typical conventions make some assumptions about the data storage which limit scale-out options.
+Initially you’ll start with all tenants placed on a single database node. To be able to scale out django, some simple changes will have to be made to your models.
 
-In particular, the ORM introduces a pattern where you normalize data and split it into many distinct models each identified by a single ``id`` column (usually added implicitly by the ORM). For instance, consider this simplified model:
+Let's consider this simplified model:
 
 .. code-block:: python
 
   from django.utils import timezone
   from django.db import models
 
-  class Store(models.Model):
-    name = models.CharField(max_length=255)
-    url = models.URLField()
+  class Country(models.Model):
+      name = models.CharField(max_length=255)
+
+  class Account(models.Model):
+      name = models.CharField(max_length=255)
+      domain = models.CharField(max_length=255)
+      subdomain = models.CharField(max_length=255)
+      country = models.ForeignKey(Country, on_delete=models.CASCADE)
```

Comment:
This isn't really related to what you're documenting, but... I wonder whether "restrict" would be a safer choice here than "cascade," since otherwise accidentally deleting a country could blow away a bunch of accounts.